ros/openapi/openapi.json

```diff
@@ -605,11 +608,9 @@
                         "type": "string",
                         "enum": [
                             "Idling",
-                            "Oversized",
                             "Undersized",
                             "Optimized",
                             "Under pressure",
-                            "Storage rightsizing",
                             "Waiting for data"
                         ]
```

Comment:
@kgaikwad Shall I remove "Undersized" as well?

Reply:
In my opinion, we can keep it as it is.